etna/transforms/feature_selection/feature_importance.py

```diff
@@ -94,3 +98,66 @@ def fit(self, df: pd.DataFrame) -> "TreeFeatureSelectionTransform":
         weights = self._get_regressors_weights(df)
         self.selected_regressors = self._select_top_k_regressors(weights, self.top_k)
         return self
+
+
+class MRMRFeatureSelectionTransform(BaseFeatureSelectionTransform):
+    """Transform that selects regressors according to MRMR variable selection method."""
+
+    def __init__(
+        self,
+        relevance_table: RelevanceTable,
+        top_k: int,
+        relevance_aggregation_mode: str = AggregationMode.mean,
+        redundancy_aggregation_mode: str = AggregationMode.mean,
+        **relevance_params,
+    ):
+        """
+        Init MRMRFeatureSelectionTransform.
+
+        Parameters
+        ----------
+        relevance_table:
+            method to calculate relevance table
+        top_k:
+            num of regressors to select; if there are not enough regressors, then all will be selected
+        relevance_aggregation_mode:
+            the method for relevance values per-segment aggregation
+        redundancy_aggregation_mode:
+            the method for redundancy values per-segment aggregation
+        """
+        if not isinstance(top_k, int) or top_k < 0:
+            raise ValueError("Parameter top_k should be positive integer")
+
+        self.relevance_table = relevance_table
+        self.top_k = top_k
+        self.relevance_aggregation_mode = relevance_aggregation_mode
+        self.redundancy_aggregation_mode = redundancy_aggregation_mode
+        self.relevance_params = relevance_params
+        self.selected_regressors: Optional[List[str]] = None
+
+    def fit(self, df: pd.DataFrame) -> "MRMRFeatureSelectionTransform":
+        """
+        Fit the method and remember features to select.
+
+        Parameters
+        ----------
+        df:
+            dataframe with all segments data
+
+        Returns
+        -------
+        result: MRMRFeatureSelectionTransform
+            instance after fitting
+        """
+        ts = TSDataset(df=df, freq=pd.infer_freq(df.index))
+        relevance_table = self.relevance_table(ts[:, :, "target"], ts[:, :, ts.regressors], **self.relevance_params)
+        if not self.relevance_table.greater_is_better:
+            relevance_table *= -1
+        self.selected_regressors = mrmr(
+            relevance_table=relevance_table,
+            regressors=ts[:, :, ts.regressors],
+            top_k=self.top_k,
+            relevance_aggregation_mode=self.relevance_aggregation_mode,
+            redundancy_aggregation_mode=self.redundancy_aggregation_mode,
```

Comment:
the atol parameter is not used in any way, can it be removed from the mrmr function?

Reply:
It can't be removed because we use it calculations, the possible solutions are:
1. Pass it as a parameter to the transform
2. Create a constant for it(in the early implementation we decided to avoid using it as a constant and put it in the list of parameters)